filebeat/input/filestream/internal/input-logfile/store.go

```diff
@@ -31,6 +32,13 @@ import (
 	"github.com/elastic/go-concert/unison"
 )
 
+// sourceStore is a store which can access resources using the Source
+// from an input.
+type sourceStore struct {
+	identifier sourceIder
+	store      *store
+}
```

Comment:
I did not put `sourceIdentifier` into `store` because if I understood correctly an `InputManager` can create multiple `Input`s of the same type with different IDs. In this case, every store reference requires a different `sourceIdentifier` instance.

Reply:
You are right.

I feel quite easily for this one :) 
I wonder if we can make this more clear in code. Maybe by renaming "type store" into "type sharedStore"?

Maybe add a godoc about the reasoning.